com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -37,6 +37,14 @@ public class MaterialUpgrader
         string m_OldShader;
         string m_NewShader;
 
+        /// <summary>
+        /// Retrieves path to new shader.
+        /// </summary>
+        public string NewShader
```

Comment:
```suggestion
        public string NewShaderPath
``` 
?

Reply:
Done in 3fa71b7caa76812686402193099bf796e231609f.